File: Python/sesion_03/3_nested_conditional.py
# Nested conditional

note = 4

message = ''

if note >= 0 and note < 5:
  message = 'You\'re suspended'
elif note >= 5 and note < 6:
  message = 'Enough'
elif note >= 6 and note < 7:
  message = 'Good'
elif note >= 7 and note < 9:
  message = 'Notable!'
elif note >= 9 and note <= 10:
  message = 'Excelent!'
else:
  message = 'Invalid note'

print(message)

File: deCero/javascript/sesion03/3_nestedConditional.js
// Nested conditional

let note = -2.6;
let message = '';

if(note >= 0 && note <= 5){
  message = 'You failed..';
}else if(note >= 5 && note <= 10){
  message = 'You passed!';
}else{
  message = 'Invalid note';
}

console.log(message);